### ISSUE TITLE
Invalid language error while submitting JavaScript

### ISSUE THREAD

**Issue by NagyAttila** [Kattis/kattis-cli#33]
Hey!

I was trying to submit a JavaScript solution to the _brokenswords_ problem and got an error.

```
attila@attila-desktop broken_swords % kattis brokenswords.js                
Problem: brokenswords
Language: JavaScript
Files: brokenswords.js
Submit (y/N)?
y
Invalid language

attila@attila-desktop broken_swords % 
```

The language is correctly detected as well as the problem, but after submission I got the 'Invalid language' error.

Explicitly setting the language to something else, for instance C, works as expected.
```
attila@attila-desktop [130] broken_swords % kattis -l c -p brokenswords brokenswords.js
Problem: brokenswords
Language: c
Files: brokenswords.js
Submit (y/N)?
y
Submission received. Submission ID: 7267240.
Open in browser (y/N)?
n
attila@attila-desktop broken_swords % 
```

Submitting on the website provides two options for Java Script: Node.js and SpiderMonkey.
I suspect that that has something to do with the problem.

Is there a way to set Node.js explicitly when submitting a solution?

**Comment by asteriskman7** [comment]
I ran into this error today also. I was able to work around it by changing the name of the language in _LANGUAGE_GUESS from "JavaScript" to "JavaScript (Node.js)". I assume you could achieve the same effect by using passing "JavaScript (Node.js)" to the -l flag on the command line.

The fix is obvious except for the fact that some users will want node.js and some will want spidermonkey.

**Comment by niemela** [comment]
This was fixed by #35.
